Base.rte/Craft/Shared/ScuttleExplosions.lua

```diff
@@ -15,7 +15,7 @@ function Update(self)
 		if self.explosionTimer:IsPastSimMS(self.explosionDelay) then
 			self.explosionTimer:Reset();
 			local explosion = CreateAEmitter("Scuttle Explosion");
-			explosion.Pos = self.Pos + Vector(self.width/2 * RangeRand(-0.9, 0.9), self.height/2 * RangeRand(-0.9, 0.9)):RadRotate(self.RotAngle);
+			explosion.Pos = self.Pos + Vector(self.width/2 * RangeRand(-0.9, 0.9), self.height * 0.5 * RangeRand(-0.9, 0.9)):RadRotate(self.RotAngle);
```

Comment:
`self.width/2` but `self.height * 0.5`
Is there a reason to replace division? I feel like `/2` is a bit easier to understand than  `* 0.5` myself.

Reply:
It seems to be very slightly faster to use multiplication instead of division in lua. It's probably not really worth doing at the scale we're operating, but there's no harm in it either (I know, premature optimization, etc.).
Personally I find both options equally readable when it's something simple like /2 vs *0.5, but anything more complex can hurt readability and I wouldn't recommend it myself.